lib/classes/CssTransformator.class.js

```diff
@@ -2,7 +2,7 @@
 const _     = require('lodash');
 const debug = require('debug')("Crittr CSSTransformator");
 const log   = require('signale');
-const merge = require('deepmerge');
+const merge = require('lodash/merge');
```

Comment:
I don't see why we should replace deepmerge with lodash/merge

Reply:
Because deepmerge cannot merge puppeteer as external browser. It fails with error.

Reply:
You should have a look at [Deepmerge:ismergeableobject](https://www.npmjs.com/package/deepmerge#ismergeableobject) which should resolve your problem with puppeteer. 

I don't think it is a good idea to include a lodash merge which mutates Object itself just to exclude instances from being cloned. 

This function will enable you to exclude all instanciated objects in merge. Which is only the puppeteer instance. 

Iam not sure if that works out. You can try it. Iam working on fixing the tests.